templates/conf/fluent.conf.erb

```diff
@@ -20,6 +20,7 @@
    port "#{ENV['FLUENT_ELASTICSEARCH_PORT']}"
    scheme "#{ENV['FLUENT_ELASTICSEARCH_SCHEME'] || 'http'}"
    ssl_verify "#{ENV['FLUENT_ELASTICSEARCH_SSL_VERIFY'] || 'true'}"
+   ssl_version "#{ENV['FLUENT_ELASTICSEARCH_SSL_VERSION'] || 'TLSv1'"
```

Comment:
Does this work? You forget to add `}` to the end.

Reply:
Oof, sorry about that @repeatedly.

I've since fixed, built the image, and ran locally using minikube and a hosted elastic search to verify.

Here's relevant log snips:
```
  <match **>
    @type elasticsearch
    @id out_es
    @log_level info
    include_tag_key true
    host xxx.composedb.com
    port 29705
    scheme https
    ssl_verify false
    **ssl_version TLSv1_2**
    user minikube-logs
    password xxxxxx
    reload_connections true
    logstash_prefix logstash
    logstash_format true
    type_name fluentd
    buffer_chunk_limit 2M
    buffer_queue_limit 32
    flush_interval 5s
    max_retry_wait 30
    disable_retry_limit
    num_threads 8
  </match>
</ROOT>
2018-12-10 14:34:21 +0000 [info]: following tail of /var/log/containers/tiller-deploy-7b7c854bc-lrn9j_kube-system_tiller-aab71cfc99226438878d2fb6efb0dc27a6830bfdd154c86d577c9ebf5c78502c.log
```

Then:
```
2018-12-10 14:34:27 +0000 [info]: Connection opened to Elasticsearch cluster => {:host=>"xxx.composedb.com", :port=>29705, :scheme=>"https", :user=>"minikube-logs", :password=>"obfuscated"}
```

Verified logs streamed over as well.